evm/src/keccak/keccak_stark.rs

```diff
@@ -565,16 +608,10 @@ mod tests {
 
         let rows = stark.generate_trace_rows(vec![input.try_into().unwrap()]);
         let last_row = rows[NUM_ROUNDS - 1];
-        let mut output = Vec::new();
         let base = F::from_canonical_u64(1 << 32);
-        for x in 0..5 {
-            for y in 0..5 {
-                output.push(
-                    last_row[reg_a_prime_prime_prime(x, y)]
-                        + base * last_row[reg_a_prime_prime_prime(x, y) + 1],
-                );
-            }
-        }
+        let output = (0..INPUT_LIMBS)
```

Comment:
Should we change `INPUT_LIMBS` --> `NUM_LIMBS`? Since it's equally the number of outputs

Reply:
What do you think about `NUM_INPUTS`, to avoid confusion since there are `2*INPUT_LIMBS` 32-bit limbs?